Library/Formula/node.rb

```diff
@@ -21,7 +22,7 @@ class Node < Formula
 
   depends_on :python => :build if MacOS.version <= :snow_leopard
   depends_on "pkg-config" => :build
-  depends_on "openssl" => :optional
```

Comment:
So (again, please pardon any ignorance) but this leaves me inclined to believe that (most likely) `--without-ssl2` is now default (i.e., not an option any more, when configuring node). So, I don't see why the _homebrew_ option needs to be changed; simply stop passing this argument to the configure script?